### ISSUE TITLE
[v13.2.0] Renderer2 createElement method ignores namespace.

### ISSUE THREAD

**Issue by mgameover** [angular/angular#44883]
### Which @angular/* package(s) are the source of the bug?

core, platform-browser

### Is this a regression?

Yes

### Description

In v13.2.0, Renderer2 creates an element without namespace even if namespace has been specified. It doesn't failed with error, however further usage of such element becomes unpredictable. 
For example, the following code tries to create an svg and set position style:
```ts
const svg = this._renderer.createElement('svg', 'http://www.w3.org/2000/svg');
this._renderer.appendChild(hostElement, svg);
this._renderer.setStyle(svg, 'position', 'fixed');
```
The error occurs in the last line, since svg element has been created incorrectly. 

After quick investigation, I found that the following line has been changed: 
https://github.com/angular/angular/blob/13.2.0/packages/platform-browser/src/dom/dom_renderer.ts#L148
![image](https://user-images.githubusercontent.com/62712576/151632642-a9c8f174-f870-4ead-b13b-30de0875ec2b.png)
This change was introduced in v13.2.0, it works fine in v13.1.3

As I understood,  `NAMESPACE_URIS` contains namespace shortcuts, but what about specifying "full" namespace or any other namespace that is not in `NAMESPACE_URIS`? 

### Please provide the exception or error you saw

```true
ERROR TypeError: Cannot set properties of undefined (setting 'width')
    at DefaultDomRenderer2.setStyle (platform-browser.mjs:618:13)
    at AnimationRenderer.setStyle (animations.mjs:282:23)
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI: 13.2.0
Node: 16.13.1
Package Manager: npm 8.1.2
OS: win32 x64

Angular: 13.2.0
... animations, cdk, cli, common, compiler, compiler-cli, core
... elements, forms, language-service, localize, material
... material-moment-adapter, platform-browser
... platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1302.0
@angular-devkit/build-angular   13.2.0
@angular-devkit/core            13.2.0
@angular-devkit/schematics      13.2.0
@schematics/angular             13.2.0
ng-packagr                      13.2.0
rxjs                            6.6.7
typescript                      4.5.5
```

**Comment by atscott** [comment]
The ` || namespace` was added in 2b9cc8503d48173492c29f5a271b61126104fbdb to specifically support how _Ivy_ was passing around namespaces, but not necessarily to add the feature of supporting arbitrary namespace URIs. Importantly, this change was not extended to `setAttribute`, `removeAttribute`, or the `ServerRenderer` so I'm pretty confident that this wasn't intended as a feature.

Given that this change was made in a minor release, we'll have to add the `|| namespace` back in to avoid a breaking change. That said, I still think it's likely that it'll be removed again in version 14 as a breaking change.

**Comment by atscott** [comment]
@mgameover There's really probably no action for you to take at the moment. I can't make recommendations on speculative changes. Either way, we'd likely write a migration at this point if we removed support for one.

**Comment by AndrewKushnir** [comment]
This issue is resolved by #44914 and the change will be available in v13.2.1 release.
